src/core/server/saved_objects/migrations/core/migration_logger.ts

```diff
@@ -32,13 +34,13 @@ export interface SavedObjectsMigrationLogger {
 }
 
 export class MigrationLogger implements SavedObjectsMigrationLogger {
-  private log: LogFn;
+  private logger: Logger;
 
-  constructor(log: LogFn) {
-    this.log = log;
+  constructor(log: Logger) {
+    this.logger = log;
   }
 
-  public info = (msg: string) => this.log(['info', 'migrations'], msg);
-  public debug = (msg: string) => this.log(['debug', 'migrations'], msg);
-  public warning = (msg: string) => this.log(['warning', 'migrations'], msg);
+  public info = (msg: string) => this.logger.info(msg);
```

Comment:
do we still need `MigrationLogger` wrapper?  now it has the same interface as Logger

Reply:
It's *almost* the same, but uses `warning()` instead of Core Logger's `warn()`. We should definitely remove this wrapper eventually, and it's probably worth just updating all existing migrations to use the new interface. I want to unblock the Saved Objects client, so I'll create a task in https://github.com/elastic/kibana/issues/33587 then we can do it as part of exposing Saved Object Migrations from Core.